server/views/pages/event.njk

```diff
@@ -24,58 +24,69 @@
       {% componentV2 'tags', tags %}
     </div>
 
-    <div class="exhibition-hero">
+
+    <div class="exhibition-hero exhibition-hero--restrict-width">
       {% componentV2 'picture', {
-          images: [{
-            contentUrl: 'https://a.travel-assets.com/findyours-php/viewfinder/images/res60/241000/241872-New-Museum.jpg',
-            type: 'picture',
-            width: 3200,
-            height: 1490
-          }]
-        }, {}, {'lazyload': true, extraClasses: ['exhibition-hero__picture']} %}
+        images: [event.featuredImage]
+      }, {}, {'lazyload': true, extraClasses: ['exhibition-hero__picture']} %}
 
       <div class="exhibition-hero__copy {{ {l:10} | spacingClasses({margin: ['bottom']}) }}">
+        <div class="is-hidden-m is-hidden-l is-hidden-xl">
+          {% component 'wobbly-edge', {background: 'white'} %}
+        </div>
+
         <div class="container {{ {l:0} | spacingClasses({padding: ['left']}) }}">
           <div class="grid">
             <div class="{{ {s:12, m:10, shiftM:1, l:12, xl:12} | gridClasses }}">
               <div class="bg-white inline-block exhibition-hero__box {{ {m:3, l:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }} rounded-diagonal">
```

Comment:
I think this `m:3` is adding padding that we don't want at this breakpoint

![screen shot 2017-10-09 at 14 11 00](https://user-images.githubusercontent.com/1394592/31339716-b12afdbc-acfb-11e7-8ee9-8ec93cbb3fac.png)

Reply:
Good catch. I've been tempted to use the classnames only to make sure we're mobile first, but think it's best tried on an evergreen project that doesn't have any implementation. Will fix